tools/sgen/input/FluentUIStyle.yml

```diff
@@ -137,6 +137,46 @@ Opacity:
   opacity96: 0.96f
   none: 1f
 
+AP_AvatarTokens:
+  backgroundDefaultColor: "#EDACB1"
+  borderRadius: [ xSmall: $Border.radius.none, small: $Border.radius.none, medium: $Border.radius.none, large: $Border.radius.none, xlarge: $Border.radius.none, xxlarge: $Border.radius.none ]
+  foregroundDefaultColor: "#6E0911"
+  presenceIconSize: [ xSmall: 0pt, small: $Icon.size.xxxSmall, medium: $Icon.size.xxxSmall, large: $Icon.size.xxSmall, xlarge: $Icon.size.xSmall, xxlarge: $Icon.size.small ]
+  presenceIconOutlineColor: $Colors.Background.neutral1
+  presenceIconOutlineThickness: [ xSmall: $Border.size.none, small: $Border.size.thick, medium: $Border.size.thick, large: $Border.size.thick, xlarge: $Border.size.thick, xxlarge: $Border.size.thick ]
+  ringDefaultColor: "#C50F1F"
+  ringGapColor: $Colors.Background.neutral1
+  ringInnerGap: [ xSmall: $Border.size.thick, small: $Border.size.thick, medium: $Border.size.thick, large: $Border.size.thick, xlarge: $Border.size.thick, xxlarge: $Border.size.thicker ]
+  ringThickness: [ xSmall: $Border.size.thin, small: $Border.size.thin, medium: $Border.size.thick, large: $Border.size.thick, xlarge: $Border.size.thick, xxlarge: $Border.size.thicker ]
+  ringOuterGap: [ xSmall: $Border.size.thick, small: $Border.size.thick, medium: $Border.size.thick, large: $Border.size.thick, xlarge: $Border.size.thick, xxlarge: $Border.size.thicker ]
+  size: [ xSmall: 16pt, small: 24pt, medium: 32pt, large: 40pt, xlarge: 52pt, xxlarge: 72pt ]
+  textFont: [ xSmall: "Font(9px, regular)", small: "Font(12px, regular)", medium: "Font(13px, regular)", large: "Font(15px, regular)", xlarge: "Font(20px, regular)", xxlarge: "Font(28px, medium)" ]
+
+NoRingAvatarTokens extends AvatarTokens:
+  ringInnerGap: [ xSmall: $Border.size.none, small: $Border.size.none, medium: $Border.size.none, large: $Border.size.none, xlarge: $Border.size.none, xxlarge: $Border.size.none ]
+  ringThickness: [ xSmall: $Border.size.none, small: $Border.size.none, medium: $Border.size.none, large: $Border.size.none, xlarge: $Border.size.none, xxlarge: $Border.size.none ]
+  ringOuterGap: [ xSmall: $Border.size.none, small: $Border.size.none, medium: $Border.size.none, large: $Border.size.none, xlarge: $Border.size.none, xxlarge: $Border.size.none ]
+  ringDefaultColor: $Colors.Neutral.clear
+
+UnauthenticatedAvatarTokens extends AvatarTokens:
```

Comment:
can we not say unauthenticated?

Reply:
This is the variation naming convention defined by the design team. I'll follow up with them on changing this convention.

Reply:
@harrieshin and @Saadnajmi. Any suggestions on how this should be named?

Reply:
How about `Anonymous` ?

Reply:
it isn't really inactive or anonymous either. it can be used with phone number only accounts, signed out state etc. I think fallback seems like a better naming?

Reply:
I'll pass the suggestions to the design team!

Reply:
@harrieshin I think "signed out" and phone only accounts should look different in the UI because a user identified only by his phone number is different from the absence of user info. 

I believe this variation is meant to be used for the signed out state (or some semantic similar to that where there is no user being represented).

Thoughts?

Reply:
@rdeassis office uses the "fallback" cases rather than "unauthenticated" style because the gray doesn't work for certain scenarios. Then, it causes more confusion why UI naming is different from usage. from previous commit of the fluent avatar, the "unauthenticated" naming was purposely not used in the control because it had too much implication on the identity state which should be the factor of UI component.

Reply:
u can easily imagine that the grey style could be used for "disabled" state for example

Reply:
Just issued PR #346 to implement the solution we discussed offline:

 - Presence of the ring is not a style. It's a state property that can be set.
 - Defined styles for the Fallback scenarios supported by the current AvatarView implementation.
 - Added the ability to set styles and sizes after the Avatar has been instantiated.
 
Issue #340 is tracking this and should be resolved by the PR. 
@Victa FYI.